packages/SwingSet/test/virtualObjects/test-representatives.js

```diff
@@ -1,7 +1,8 @@
 /* global __dirname */
-import '@agoric/install-ses';
+import { test } from '../../tools/prepare-test-env';
+
+// eslint-disable-next-line import/order
```

Comment:
It's curious that this lint override is necessary. I know that side-effecting imports must be separated from the others by a blank like, else `prettier` likes to alphabetize them or something (breaking the necessary ordering). But I thought it backed off and only attempted to sort imports that were in a blank-line-delimited group. Weird.

Reply:
I don't like the inconsistency either, but I don't understand it at all. This is not something I thought about. Rather, I just blindly put in these disables as vscode suggested, until all the red went away.